src/utils/Load.js

```diff
@@ -1,10 +1,11 @@
 import tarStream from 'tar-stream';
 import zlib from 'zlib';
 import http from 'http';
+import { logger } from 'fsh-sushi/dist/utils';
```

Comment:
Sorry I didn't notice this before, but I think we usually try to import things from fsh-sushi without using the dist folder. So similar to how we have it in RunSUSHI.js, I think can do something like

```
import { utils } from 'fsh-sushi';
const logger = utils.logger;
```

That lets us get the utils export and then get the logger from that.